DEVELOPMENT.md

```diff
@@ -94,7 +94,7 @@ source ./hack/test-env.sh
 #### Install Gateway API CRDs
 
 ```
-kubectl apply -k "github.com/kubernetes-sigs/gateway-api/config/crd?ref=${GATEWAY_API_VERSION}"
+kubectl apply -f config/100-gateway-api.yaml
```

Comment:
It feels strange to be copying the definitions into our repo, rather than referencing some upstream artifact (even if it's a raw or nightly reference rather than a release).

Reply:
It feels practical to me rather than strange. If I'm using artifacts to change/test the code, I'd rather have it sitting on my machine rather than going over the network. And in this case I can control/change the version I want by changing the version variable the codegen uses. Fetch once/run locally with the same copy. Is this an anti-pattern?